common/src/test/java/io/pravega/schemaregistry/common/FuturesUtilityTest.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.common;
+
+import com.google.common.collect.Lists;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.AbstractMap;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.function.BiFunction;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class FuturesUtilityTest {
+
+    private ScheduledExecutorService executor;
+    @Before
+    public void setUp() {
+        executor = Executors.newSingleThreadScheduledExecutor();
+    }
+    
+    @After
+    public void tearDown() {
+        executor.shutdownNow();    
```

Comment:
Please call `io.pravega.common.concurrent.ExecutorServiceHelpers.shutdown(..)` to cleanly shutdown the executor, which uses the 2-phase shutdown protocol described in [this ](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ExecutorService.html#:~:text=Two%20different%20methods%20are%20provided,to%20stop%20currently%20executing%20tasks.)Oracle documentation.